src/main/resources/hapi.properties

```diff
@@ -31,14 +31,14 @@ logger.name=fhirtest.access
 logger.format=Path[${servletPath}] Source[${requestHeader.x-forwarded-for}] Operation[${operationType} ${operationName} ${idOrResourceName}] UA[${requestHeader.user-agent}] Params[${requestParameters}] ResponseEncoding[${responseEncodingNoDefault}]
 logger.error_format=ERROR - ${requestVerb} ${requestUrl}
 logger.log_exceptions=true
-datasource.driver=org.apache.derby.jdbc.EmbeddedDriver
-datasource.url=jdbc:derby:directory:target/jpaserver_derby_files;create=true
-datasource.username=
-datasource.password=
+datasource.driver=com.mysql.cj.jdbc.Driver
```

Comment:
I would rather leave the default configuration using Derby since that will work for anyone regardless of what other software they have installed.

Reply:
Noted, have reverted back to derby and instead added some updates on readme regarding mysql usage.